### ISSUE TITLE
Add commit history tree color settings/options

### ISSUE THREAD

**Issue by NordicAngel** [GitFiend/Support#179]
I have been checking out GitFiend and like it so far. Though I would suggest a way of customizing the look of the commit tree. As far as I can tell the current colors are per commit and based on the user who made the commit. I would like a way of seeing the tree color coordinated based on branch color and maybe a way of highlighting a specific branch. I wanna specific this as an option as I also like the current view.

**Comment by NordicAngel** [comment]
I am referring to the commit graph when i say 'commit tree'.

**Comment by GitFiend** [comment]
Yeah, I thought about doing this initially. I didn't think of a nice way to colour by branch as branches often share a lot of commits. I like the second idea of highlighting a branch/path through you can select.
